modules/waf/README.md

```diff
@@ -0,0 +1,38 @@
+# CIS web application firewall (WAF) module
+
+This module enables/disables the web application firewall (WAF) of the domain.
+
+> When the `waf` is enabled then the `min_tls_version` is automatically modified to `1.1`.
```

Comment:
whats the purpose to have this line here?

Reply:
I was working on multiple branches so by mistake I have added it here.